src/adapters/v00/marketplace_adapter.js

```diff
@@ -1,3 +1,5 @@
+const OFFER_STATUS = ['error','created', 'accepted', 'finalized', 'buyerReviewed']
```

Comment:
Changes in this file must also be applied to src/adapters/v01/marketplace_adapter.js